server/roomlogs.ts

```diff
@@ -211,8 +217,25 @@ export class Roomlog {
 				break;
 			}
 		}
-		this.broadcastBuffer += message + '\n';
-		return this;
+	}
+	addUserUhtmlchange(user: User, name: string, message: string) {
```

Comment:
This should just be named `uhtmlchange` - the old `uhtmlchange` isn't used anywhere, is it?

If it is, I would call this `uhtmlchangeByUser` or something, so they make sense.

Reply:
The `uhtmlchange` method is used in `add` method.

I don't think `uhtmlchangeByUser` is a good name for method, considering it's not working by user, rather it works on attributed `uhtmlchange`. Maybe I should name the method `addAttributedUhtmlchange` instead.

Note that `uhtmlchange` is distinct from this method, as it doesn't actually `add`.